Plugins/Disk/Retrieve-FLSBody.ps1

```diff
@@ -87,7 +87,15 @@ process{
     }
 
     #Run on the remote host and collect data
-    $ScriptBlock = $ExecutionContext.InvokeCommand.NewScriptBlock(("& '{0}\fls\fls.exe' -r -m '{1}' '\\.\{1}' | Out-File '{0}\FLS.body'") -f ($RemotePath,$SystemDrive))
+    $ScriptBlock = $ExecutionContext.InvokeCommand.NewScriptBlock(("& '{0}\fls\fls.exe' -r -m '{1}' '\\.\{1}' | Out-File '{0}\FLSi.body'") -f ($RemotePath,$SystemDrive))
+    
+    Invoke-Command -Session $Session -ScriptBlock $ScriptBlock
+
+    #Format the FLS body file so that it works with all operating systems and not just Windows because CRLF 
+
+    $ScriptBlock = $ExecutionContext.InvokeCommand.NewScriptBlock(("Get-Content '{0}\FLSi.body' -raw | ForEach-Object {{`$_ -replace `"``r`", `"`"}} | Set-Content -NoNewLine '{0}\FLS.body'") -f ($RemotePath))
+
+    Write-Debug "Scriptblock"

```

Comment:
Did you want to keep this `Write-Debug` statement in here?

Reply:
fixed.